coalib/output/ConsoleInteraction.py

```diff
@@ -172,7 +172,7 @@ def print_lines(console_printer,
                               color=FILE_LINES_COLOR,
                               end='')
 
-        line = file_dict[sourcerange.file][i - 1].rstrip("\n")
+        line = file_dict[sourcerange.file].lines[i - 1].rstrip("\n")
```

Comment:
I think this is a fairly common usecase. Can FileProxy have a helper function to remove newlines ? That would make it more elegant to use.

So, essentially have a function `.lines(start=0, end=-1, with_newline=False)` This way we can even handle the splicing inside the function. There would be a similar function `.line(line_num, with_newline=False)` for a single line.

Maybe I am over thinking and over wrapping the File though ... opinions welcome.